tests/utils/formatDate.test.ts

```diff
@@ -88,3 +95,28 @@ describe('Formatting dates', () => {
     expect(formatted).toEqual('9/27/2013')
   })
 })
+
+// Test formatFromApiGateway()
+describe('Requesting a date adjusted from existing in the correct format', () => {
+  it('has our time set to PT!', () => {
+    expect(DateTime.local().zoneName).toEqual('America/Los_Angeles')
```

Comment:
Question about luxon: Do you need to also add `Settings.defaultZone = 'America/Los_Angeles'` to this file? How does `DateTime` now which timezone is local?

Reply:
Good question! I tested adding it in various places. Initially I was hopeful adding it to the ServerSide function in `page/index.tsx` would be sufficient, but no. Then I added it to here, also didn't work, and then I added it to utils, and it does work. I believe the Setting affects the behavior everywhere once its set

I think if I were more JS-Server-Stuff savvy I would know a better place to put it 🤔

Reply:
I feel a little surprised by that behavior 🤔  but 🤷